### ISSUE TITLE
Suggestions while adding new tag to logic hidden

### ISSUE THREAD

**Issue by chandansgowda** [Hackathon-Eigen/hackathon_project#41]
When we are trying to add a tag to the logic of the group, the tag name suggestions are hidden behind the keyboard as in the attached screenshot.

![suggestions_hidden](https://user-images.githubusercontent.com/41890434/208280219-d87f93d1-9b94-416f-ac95-c4e4b25d535c.png)

**Comment by letsintegreat** [comment]
Isn't that list scrollable?

**Comment by chandansgowda** [comment]
Yes. It's scrollable but the user experience in devices with lesser height will not be good.

**Comment by letsintegreat** [comment]
@chandansgowda Alright, what solution are you proposing?

**Comment by chandansgowda** [comment]
@letsintegreat The suggestions can come above the textfield or the whole widget can be refactored into a new screen.
